react/src/components/organisms/Teaser/storyutils.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
```

Comment:
should any of these be pulled out into a higher utils folder?

Reply:
In this case I want to say no; If while working on another component you find you need them, then they should be pulled out, but not before. A lot of the options are pretty specific to the story for this Teaser component.